src/app.js

```diff
@@ -1,88 +1,167 @@
-var nav = document.createElement("div")
-nav.classList.add("m-3")
+const content = document.getElementById('content');
 
-var h1 = document.createElement('h1');
+const nav = document.createElement('div');
+nav.classList.add('m-3');
 
-var ul = document.createElement("ul")
-ul.classList.add("navbar-nav")
-nav.classList.add("d-flex")
-ul.classList.add("d-flex")
-ul.classList.add("flex-row")
-nav.classList.add("justify-content-between")
-nav.classList.add("align-self-center")
+const h1 = document.createElement('h1');
 
+const ul = document.createElement('ul');
+ul.classList.add('navbar-nav');
+nav.classList.add('d-flex');
+ul.classList.add('d-flex');
+ul.classList.add('flex-row');
+nav.classList.add('justify-content-between');
+nav.classList.add('align-self-center');
 
-var li1 = document.createElement("li")
-li1.appendChild(document.createTextNode("Menu"));
-li1.classList.add(...(["mx-5", "mt-3"]))
 
+const li1 = document.createElement('li');
+li1.appendChild(document.createTextNode('Menu'));
+li1.classList.add(...(['mx-5', 'mt-3']));
 
 
+const li2 = document.createElement('li');
+li2.appendChild(document.createTextNode('Contact'));
+li2.classList.add('mt-3');
 
 
-var li2 = document.createElement("li")
-li2.appendChild(document.createTextNode("Contact"));
-li2.classList.add("mt-3")
-
-
-ul.appendChild(li1)
-ul.appendChild(li2)
+ul.appendChild(li1);
+ul.appendChild(li2);
 
+content.appendChild(nav);
+h1.appendChild(document.createTextNode('The Restaurant'));
+h1.onclick = function () {
+  window.location.reload(true);
+};
 
-var content = document.getElementById("content")
 
-content.appendChild(nav)
-h1.appendChild(document.createTextNode("The best food in the world"));
-h1.onclick = function() {
-	window.location.reload(true)
+nav.appendChild(h1);
+nav.appendChild(ul);
+
+function createDiv() {
```

Comment:
- [ ] You should use **arrow functions** here instead of **traditional keywords** to be compliant with `ES6` standards.